### ISSUE TITLE
Use the new context menu API

### ISSUE THREAD

**Issue by Croydon** [Croydon/vertical-tabs-reloaded#205]
With `browser.menus.overrideContext()` it is now possible again to use native context menus with unlimited custom entries + other add-ons can display their entries there as well. Finally!

This issue is about migration the custom context menu to the new extension APIs.

**Comment by grahamperrin** [comment]
Hi, are there any show-stoppers to this?

**Comment by Croydon** [comment]
No, I just didn't come around to work on VTR in a long time, but hopefully I can make time again in the next weeks 😄
